src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -146,15 +146,15 @@ test_swim_status_handler(crt_rpc_t *rpc_req)
 	regex_t				 regex_dead;
 	static const char		*dead_regex = ".?0*1";
 	static const char		*alive_regex = ".?0*";
-	char				*swim_seq = malloc(MAX_SWIM_STATUSES);
+	char				swim_seq[MAX_SWIM_STATUSES];
 
 	/* CaRT internally already allocated the input/output buffer */
 	e_req = crt_req_get(rpc_req);
 
 	if (swim_status_by_rank[e_req->rank] != NULL)
 		strcpy(swim_seq, swim_status_by_rank[e_req->rank]);
```

Comment:
Since this is now a fixed buffer this should be strncpy instead.

Reply:
Can we use `memcpy` instead of `strncpy`?  In a [previous coverity fix](https://jira.hpdd.intel.com/browse/DAOS-6844), I think we hit an issue with `strncpy` on Ubuntu: https://github.com/daos-stack/daos/pull/4762/commits/1b9cf80ab23dfd52f1ed4b5e25ca4bdf7e2f3b04.

Reply:
memcpy is fine as well.

Reply:
d2e396146e575c90f1b16ede9d887b5c5c5c8ebc